x-pack/legacy/plugins/siem/server/lib/framework/types.ts

```diff
@@ -37,27 +36,17 @@ export interface FrameworkAdapter {
     method: 'msearch',
     options?: object
   ): Promise<DatabaseMultiResponse<Hit, Aggregation>>;
-  callWithRequest(
```

Comment:
we were thinking to use that in DetetcionEngine. Did we remove it because we are not using it?

Reply:
Yep! As mentioned in today's sync, this was [implemented in the detection engine](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/utils.ts#L99-L104) for now. I'd like to consolidate these different calls again, but that'll be easier once detection engine is also on the new platform.